CMakeLists.txt

```diff
@@ -16,13 +16,15 @@ find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rcutils)
 find_package(builtin_interfaces REQUIRED)
+find_package(std_msgs REQUIRED)
```

Comment:
I only see this being used in a test https://github.com/ros2/message_filters/pull/7/files#diff-d26d660e83afc31871c6cfeaecd487e8R40 So this should be in a testing only block.